packages/features/bookings/schemas/bookingReadPublic.schema.ts

```diff
@@ -0,0 +1,57 @@
+import { z } from "zod";
+
+import { _AttendeeModel, _BookingModel as Booking, _PaymentModel, _UserModel } from "@calcom/prisma/zod";
+import { EventTypeMetaDataSchema } from "@calcom/prisma/zod-utils";
+
+export const bookingReadPublicSchema = Booking.extend({
```

Comment:
Taken from API libs.

The bad news is that we need to set and output schema for EVERY single tRPC procedure that we want to include.

The good news is that we already have most output schemas in the API libs.

Reply:
I'm up for this, it will make testing even easier I think.